python/pyspark/pandas/groupby.py

```diff
@@ -581,6 +581,64 @@ def mean(self, numeric_only: Optional[bool] = True) -> FrameLike:
             F.mean, accepted_spark_types=(NumericType,), bool_to_numeric=True
         )
 
+    # TODO: 'q' accepts list like type
+    def quantile(self, q: float = 0.5, accuracy: int = 10000) -> FrameLike:
+        """
+        Return group values at the given quantile.
+
+        .. note:: `quantile` in pandas-on-Spark are using distributed percentile approximation
+            algorithm unlike pandas, the result might different with pandas in accuracy, also
+            `interpolation` parameters are not supported yet.
+
+        Parameters
+        ----------
+        q : float, default 0.5 (50% quantile)
+            Value between 0 and 1 providing the quantile to compute.
+        accuracy : int, optional
+            Default accuracy of approximation. Larger value means better accuracy.
+            The relative error can be deduced by 1.0 / accuracy.
+            This is a panda-on-Spark specific parameter.
+
+            .. versionadded:: 3.4.0
```

Comment:
Seems like this should be placed the above `Parameters`, since this whole API is newly implemented, not only parameters.

e.g.
```
    def quantile(self, q: float = 0.5, accuracy: int = 10000) -> FrameLike:
        """
        Return group values at the given quantile.

        .. versionadded:: 3.4.0

        Parameters
        ----------
```